### PR TITLE
Rich Text Editor

### PR DESCRIPTION
**Pull request by horsemaker**
- Added Notes Screen
- Added Editor Constants
- Added Note Details Reducer
- Added Rich Text Editor Component

### CODE REVIEW

File: src/frontend/components/RichTextEditor/RichTextEditor.css [hunk 1]
```diff
@@ -0,0 +1,289 @@
+.rich-text-editor {
+  max-width: calc(100vw - 2rem);
```
Comment (vijaytembugade) [inline src/frontend/components/RichTextEditor/RichTextEditor.css:2]: This is good. Thanks .

**Review by vijaytembugade — APPROVED** [review]
code looks clean and Nice editor. 👍

File: src/frontend/components/RichTextEditor/RichTextEditor.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                onClick={() =>
+                  dispatchNoteDetails({ type: COLOR, payload: "color-2" })
+                }
+              ></span>
+              <span
+                className={
+                  color === "color-3" ? "color-3 color-outline" : "color-3 "
+                }
+                onClick={() =>
+                  dispatchNoteDetails({ type: COLOR, payload: "color-3" })
```
Comment (narharikale) [inline src/frontend/components/RichTextEditor/RichTextEditor.js:136]: liked the implementation of color labels ✨

**Review by narharikale — APPROVED** [review]
awesome UI 🔥 and code is clean

**Pull request merged**